src/Components/SynthEngine/SynthEngine.js

```diff
@@ -0,0 +1,58 @@
+import * as Tone from 'tone';
```

Comment:
I like the separation of concern here in having the synth parts in its own file.